packages/js/src/generators/library/library.ts

```diff
@@ -196,9 +231,11 @@ function createFiles(tree: Tree, options: NormalizedSchema, filesDir: string) {
     hasUnitTestRunner: options.unitTestRunner !== 'none',
   });
 
-  if (options.buildable && options.compiler === 'swc') {
+  if (options.compiler === 'swc') {
     addSwcDependencies(tree);
     addSwcConfig(tree, options.projectRoot);
+  } else if (shouldAddBabelRc(tree, options)) {
+    addBabelRc(tree, options);
```

Comment:
@nartc not super familiar with SWC outside its usage from next@12, my assumption is that if you explicitly set you library to be compiled in `swc`, there won't be any scenario where you _also_ would need a `.babelrc` in the project?

With my implementation, if you use `--includeBabelRc=true` and you also have `--compiler=swc` then you won't generate a `.babelrc` file

Reply:
Yeah. That should be good.